README.md

```diff
@@ -90,6 +91,24 @@ Executables here will run just before those in `/opt/odoo/common/build.d`.
 Files here will be environment-variable-expanded and concatenated in
 `/opt/odoo/auto/odoo.conf` at build time.
 
+#### `/opt/odoo/custom/ssh.d`
+
+Files here will be used to create the `root` and `odoo` users' SSH directories.
+
+It should follow the same structure as a standard `~/.ssh` directory, including
+`config` and `known_hosts` files.
+
+The `config` file should contain `IdentityFile` keys to represent the private
+key that should be used for that host. The key will be located in  `~/.ssh/`.
```

Comment:
You could add some docs links on how to use the ssh_config file.

Also some FAQs to document answer to @pedrobaeza's concerns and to document why it's more secure to hardcode the git server identity, based on your comments too.